.github/CONTRIBUTING.md

```diff
@@ -94,6 +127,52 @@ mlc docs/
 
 It only checks links within the local docs (it can't check links to other docs sections that are present when we publish all products under https://redocly.com/docs), and doesn't currently check anchors, so take care when renaming pages or titles.
 
+### Built-in rules changes
+
+After adding a new rule, make sure it is added to the `minimal`, `recommended` and `all` rulesets with appropriate severity levels.
+Also add the rule to the `builtInRulesList` in [the config types tree](../packages/core/src/types/redocly-yaml.ts).
+
+Separately, open a merge request with the corresponding documentation changes.
```

Comment:
An added rule will land after the next release, whilst the docs will become accessible almost immediately after merging. This could lead to users seeing the docs, but the rule doesn't work. That's why having it separately and merging it alongside the release is better. We should link the MRs together via references though.